src/utils/getWeatherData.js

```diff
@@ -18,7 +16,9 @@ const getWeatherData = async (location, setWeatherData) => {
       setWeatherData(value);
     })
     .catch((error) => {
-      console.log("Something went wrong ! Refresh the page after 1 min");
+      console.log(
```

Comment:
suggestion: you can use console.error(), so error would be displayed in error section separately in the console of browser dev tool.

Reply:
Done !